docs/server/source/cloud-deployment-starter-templates/new-azure-template.md

```diff
@@ -0,0 +1,57 @@
+
+One can deploy a BigchainDB node on Azure using the template in the `bigchaindb-on-ubuntu` directory of Microsoft's `azure-quickstart-templates` repository on GitHub:
+
+1. Go to [the `bigchaindb-on-ubuntu` directory in that repository](https://github.com/Azure/azure-quickstart-templates/tree/master/bigchaindb-on-ubuntu).
+
+2. Click the button labelled **Deploy to Azure**.
+
+3. If you're not already logged in to Microsoft Azure, then you'll be prompted to login. If you don't have an account, then you'll have to create one.
+
+4. One you are logged in to the Microsoft Azure Portal, you should be taken to a form titled **BigchainDB**. There are some notes to help with filling in that form at the bottom of this page.
+
+5. Deployment takes a few minutes. You can follow the notifications by clicking the bell icon at the top of the screen. When done, you should see a notification saying "Deployment to resource group '[your resource group name]' was successful." The install script (`init.sh`) installed RethinkDB, configured it using the default RethinkDB config file, and ran it. It also used `pip` to install [the latest `bigchaindb` from PyPI](https://pypi.python.org/pypi/BigchainDB).
+
+6. Find out the public IP address of the virtual machine in the Azure Portal. Example: `40.69.87.250`
+
+7. ssh in to the virtual machine at that IP address, e.g. `ssh adminusername@40.69.87.250` (where `adminusername` should be replaced by the Admin Username you entered into the form, and `40.69.87.250` should be replaced by the IP address you found in the last step).
+
+8. You should be prompted for a password. Enter the Admin Password you entered into the form.
+
+9. Configure BigchainDB using the default BigchainDB settings: `bigchaindb -y configure`
+
+10. Run BigchainDB: `bigchaindb start`
+
+BigchainDB should now be running on the Azure VM.
+
+Remember to shut everything down when you're done (via the Azure Portal), because it generally costs money to run stuff on Azure.
+
+
+## Notes on the Blockchain Template Form Fields
+
+### BASICS
+
+**Resource group** - You can use an existing resource group (if you have one) or create a new one named whatever you like, but avoid using fancy characters in the name because Azure might have problems if you do.
+
+**Location** is the Microsoft Azure data center where you want the BigchainDB node to run. Pick one close to where you are located.
+
+### SETTINGS
+
+You can use whatever **Admin\_username** and **Admin\_password** you like (provided you don't get too fancy). It will complain if your password is too simple. You'll need these later to `ssh` into the virtual machine.
+
+**Dns\_label\_prefix** - Once your virtual machine is deployed, it will have a public IP address and a DNS name (hostname) something like `DNSprefix.northeurope.cloudapp.azure.com`. The `DNSprefix` will be whatever you enter into this field.
```

Comment:
Not 100% sure what we're using as conventions for substitutable strings in the docs, but it's nice to have some sort of delimiter to visually clarify them, e.g. `<DNSprefix>.northeurope.cloudapp.azure.com`.